### ISSUE TITLE
Error at Exercise 1.1: Using the bigip_device_info module

### ISSUE THREAD

**Issue by carlosfparada** [ansible/workshops#1529]
### Area

Ansible F5 Workshop

### Proposed Changes

The EE image (https://quay.io/repository/acme_corp/security_ee?tab=tags) does not contain the f5networks.f5_modules collection.

**Comment by carlosfparada** [comment]
Digging a bit, I created a new EE with the missing collections, but still other things wrong, e.g. inventory is completely different, and some python code is failing. I noticed there is a f5_ee EE mentioned in the WS descripton and not used in the code, but even with this things fail on the python code

**Comment by anshulbehl** [comment]
Hi @carlosfparada 

The EE that works with f5 is https://quay.io/repository/acme_corp/f5_ee
This should have all the dependencies

**Comment by carlosfparada** [comment]
In fact the environment is configured (.ansible-navigator.yaml) to use the security_ee, even if in https://aap2.demoredhat.com/exercises/ansible_f5/1.0-explore/ if shows the f5ee.
Anyway, I tried the f5_ee and did not work as well, it raises some python errors.

**Comment by carlosfparada** [comment]
@anshulbehl now I understand what is happening. In fact I was using the wrong Lab. I used the security Lab to run the F5 exercises. My fault, so never mind this issue. But anyway, do you know why I do not see (and cannot order) the F5 Lab in the "Management Automation Workshops Demos" folder in RHPDS, only the other 5?

**Comment by anshulbehl** [comment]
@carlosfparada yeah the reason for that is f5 workshops still need some support from the F5 side to be maintained, we are working with f5 on that front, but it will take some time for the workshop to be fully AAP2 compatible

**Comment by carlosfparada** [comment]
OK, thanks @anshulbehl for the clarification

**Comment by IPvSean** [comment]
closing this since its been 15 days
